xwiki-platform-core/xwiki-platform-tag/xwiki-platform-tag-ui/src/main/resources/XWiki/TagCloud.xml

```diff
@@ -338,8 +347,25 @@ ol.tagCloud .ultraPopular {
 ##
 $xwiki.ssx.use('XWiki.TagCloud')##
 ##
-## Get tag count map. Key: tag, value: number of occurrences.
-#set ($tagCount = $xwiki.tag.getTagCount("$!tagCloudSpace"))
+##if (!("$!tagCloudSpace" =="") &amp;&amp; !("$!tagCloudSpaces" ==""))
+#if( $tagCloudSpace &amp;&amp; $tagCloudSpaces )
+{{html}}
```

Comment:
Of course, I was joking. Thanks for your explanation.
I'm a little bit lazy but if you show me clearly and simply how to do it I can do it :-) for error and translation macro...
I must add something like this?
<dependency>
      <groupId>org.xwiki.rendering</groupId>
      <artifactId>xwiki-rendering-macro-message</artifactId>
      <version>${project.version}</version>
      <scope>runtime</scope>
    </dependency>
<dependency>